Downloaded this binary [unimap-linux](https://github.com/Edu4rdSHL/unimap/releases/download/0.5.1/unimap-linux) When I run this it returns an error like below:

```
rudra@workspace:~$ ./unimap-linux -t example.com

[INFO] Performing parallel resolution for 1 targets with 1 threads, it will take a while...

[ERROR] Error waiting command to finish for 1xx.1x.2x.2xx, continuing with remaining hosts. Description: No such file or directory (os error 2)+------------+---------------+------------+----------+
|    HOST    |      IP       | OPEN PORTS | SERVICES |
+============+===============+============+==========+
| example.com | 1xx.1x.2x.2xx | NULL       | ++       |
|            |               |            | ++       |
+------------+---------------+------------+----------+
```

As it's not working then the user has to compile the binary and here I just added a line in README for globally using the binary comment added you can check from there.

--- a/README.md
+++ b/README.md
@@ -20,7 +20,7 @@ You need to have Rust and Nmap installed in your computer, then run:
 2. cargo build --release
 # Now the binary is in ./target/release/unimap
 3. cp ./target/release/unimap /usr/local/bin
-# copy the unimap binaries for run from everywhere
+# Now you can use the `unimap` command from everyewhere.
 ```
 
 ## Using precompiled binaries

--- a/README.md
+++ b/README.md
@@ -21,6 +21,7 @@ You need to have Rust and Nmap installed in your computer, then run:
 # Now the binary is in ./target/release/unimap
 3. cp ./target/release/unimap /usr/local/bin
 # Now you can use the `unimap` command from everyewhere.
+# This command only suuport Linux and MacOS. Windows user need specify the target path in environment variable
 ```
 
 ## Using precompiled binaries

--- a/README.md
+++ b/README.md
@@ -19,9 +19,9 @@ You need to have Rust and Nmap installed in your computer, then run:
 # extract it and continue to next step.
 2. cargo build --release
 # Now the binary is in ./target/release/unimap
+# The next command only works on Linux and MacOS. Windows user need specify the target path in environment variable
 3. cp ./target/release/unimap /usr/local/bin
 # Now you can use the `unimap` command from everyewhere.
-# This command only suuport Linux and MacOS. Windows user need specify the target path in environment variable
 ```
 
 ## Using precompiled binaries

